tutorials/data_generation_variant.ipynb

```diff
@@ -0,0 +1,572 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data preparation for missense variants"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "<img style=\"margin-left: 1.5rem\" align=\"right\" src=\"images/data_generation_variants.png\" width=\"400\">\n",
+    "\n",
+    "This tutorial will demonstrate the use of DeepRank-Core for generating missense variants graphs and saving them as [HDF5 files](https://en.wikipedia.org/wiki/Hierarchical_Data_Format) files, using [PBD files](https://en.wikipedia.org/wiki/Protein_Data_Bank_(file_format)) of protein structures as input.\n",
+    "\n",
+    "In this data processing phase, for each variant a local neighborhood around itself is selected according to a radius threshold that the user can customize, and it is mapped to a graph. Nodes either represent residues or atoms, and edges are the interactions between them. Each node and edge can have several different features, which are generated and added during the processing phase as well. Optionally, the graphs can be mapped to volumetric grids (i.e., 3D image-like representations). The mapped data are finally saved into HDF5 files, and can be used for later models' training (for details go to [training_ppi.ipynb](https://github.com/DeepRank/deeprank-core/blob/main/tutorials/training_ppi.ipynb) tutorial). In particular, graphs can be used for the training of Graph Neural Networks (GNNs), and grids can be used for the training of Convolutional Neural Networks (CNNs)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Input Data\n",
+    "\n",
+    "The example data used in this tutorial are available on Zenodo at [this record address](https://zenodo.org/record/8187806). To download the raw data used in this tutorial, please visit the link and download `data_raw.zip`. Unzip it, and save the `data_raw/` folder in the same directory as this notebook. The name and the location of the folder are optional but recommended, as they are the name and the location we will use to refer to the folder throughout the tutorial.\n",
+    "\n",
+    "Note that the dataset contains only 96 data points, which is not enough to develop an impactful predictive model, and the scope of its use is indeed only demonstrative and informative for the users."
```

Comment:
I think my download only contains 58 pdb files, but the csv has 100 entries. Not sure how those relate, but they are both different from 96 :)
I mentioned 58 a few times in other comments. If 100 is accurate, ignore/adjust what I said there ;)

Reply:
I misunderstood that one pdb can have multiple variants, so I think no change needed.
If the csv has some excess unused entries we can just leave it as is, imo.